src/PostgREST/Error.hs

```diff
@@ -228,7 +228,19 @@ checkIsFatal (PgError _ (P.ConnectionError e))
   where isAuthFailureMessage = "FATAL:  password authentication failed" `isPrefixOf` toS failureMessage
         failureMessage = fromMaybe mempty e
 -- Chek for a syntax error(42601 is the pg code). This would mean the error is on our part somehow, so we treat it as fatal.
-checkIsFatal (PgError _ (P.SessionError (H.QueryError _ _ (H.ResultError (H.ServerError "42601" e _ _))))) = Just $ toS e
+checkIsFatal (PgError _ (P.SessionError (H.QueryError _ _ (H.ResultError (H.ServerError "42601" _ _ _))))) =
+  Just "Hint: This is probably a bug in PostgREST, please report it at https://github.com/PostgREST/postgrest/issues"
+-- Check for a "prepared statement already exists" (Code 42P05: duplicate_prepared_statement).
+-- This would mean that a connection pooler in transaction mode is being used
+-- and incompatible features such as prepared statements or notification channels are enabled in the PostgREST configuration.
```

Comment:
Hm, when the channel is enabled, does that give the duplicate prepared stmt error? If not, then this comment is not precise. Do we get a particular error for the channel when connecting to the tx pooler?

Reply:
You're right, the notification channel with transaction pooling doesn't give an error. I'll fix the comment and the hint.